api/kyverno/v2/policy_exception_types.go

```diff
@@ -128,13 +128,43 @@ type Exception struct {
 
 	// RuleNames identifies the rules to which the exception is applied.
 	RuleNames []string `json:"ruleNames"`
+
+	// Images specifies image-based exceptions for verifyImages rules.
+	// This allows exempting specific images from image verification policies.
+	// +optional
+	Images []ImageException `json:"images,omitempty"`
```

Comment:
In [this](https://github.com/kyverno/kyverno/issues/8663) issue, we designed it as follows:
```
   verifyImages:
      imageReferences:
        - "docker.io/mariamfahmy98/busybox*"
 ```
So we need to re-discuss the design.